mautic/mautic
# Fix timeline pagination title

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | ✓
| New feature? | 
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | 
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:

<img width="748" alt="screen shot 2017-08-09 at 11 58 32" src="https://user-images.githubusercontent.com/18265735/29116139-351482ba-7cfa-11e7-8d51-ed60c41c52ad.png">

If a contact's timeline is paginated, clicking on one of the page links breaks the title.

[//]: # ( As applicable: )
#### Steps to reproduce the bug:
1. View a contact with a paginated timeline
2. Click on a page link
3. Notice the broken title

#### Steps to test this PR:
1. View a contact with a paginated timeline
2. Click on a page link
3. Notice the intact title
